lib/utils/extension/build_context_extension.dart

```diff
@@ -168,4 +168,20 @@ extension ContextExtensionss on BuildContext {
     );
     return position;
   }
+
+  void pushInner(String path, {Object? arguments}) {
+    Navigator.of(this).pushNamed(path, arguments: arguments);
+  }
+
+  void pushInnerReplace(String path, {Object? arguments}) {
+    Navigator.of(this).pushReplacementNamed(path, arguments: arguments);
+  }
+
+  void popInner() {
+    Navigator.of(this).pop();
+  }
+
+  void popInnerAll() {
+    Navigator.of(this).popUntil((route) => route.isFirst);
+  }
```

Comment:
@nqhhdev @drminh2807 wdyt aboout this? It can apply for every context?

Reply:
Why not a `mixin`? Only some context can do it? @sherlockvn

Reply:
Khai want to same way to call with go_router `context.go(path)`